<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commandDesignerWidget</class>
 <widget class="QWidget" name="commandDesignerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Components</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget_">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="buildCommandTab_">
           <attribute name="title">
            <string>Build command</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="MessageLabel" name="infoLabel_" native="true"/>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QListWidget" name="componentsList_">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>ecflow_client</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>&lt;node_name&gt;</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>&lt;full_name&gt;</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="commandManPage_">
                 <property name="font">
                  <font>
                   <family>Monospace</family>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="commandHelpLabel_">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="NodesTab_">
           <attribute name="title">
            <string>Selected Nodes</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="NodeQueryResultView" name="nodeSelectionView_"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="savedCommandsTab_">
           <attribute name="title">
            <string>Saved commands</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QTableWidget" name="savedCommandsTable_">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Context menu</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Command</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Command</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="commandLineEdit_"/>
          </item>
          <item>
           <widget class="QPushButton" name="runButton_">
            <property name="text">
             <string>&amp;Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="nodeListLinkLabel_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;#nodes&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Nodes selected&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="saveCommandGroupBox_">
        <property name="title">
         <string>Save command</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="saveNameLineEdit_"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="addToContextMenuCb_">
           <property name="text">
            <string>Add to context menu</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="saveAsNewButton_">
             <property name="text">
              <string>Save As &amp;New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="overwriteButton_">
             <property name="text">
              <string>&amp;Overwrite</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NodeQueryResultView</class>
   <extends>QTreeView</extends>
   <header>NodeQueryResultView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>saveNameLineEdit_</tabstop>
  <tabstop>overwriteButton_</tabstop>
  <tabstop>tabWidget_</tabstop>
  <tabstop>savedCommandsTable_</tabstop>
  <tabstop>addToContextMenuCb_</tabstop>
  <tabstop>saveAsNewButton_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
